### ISSUE TITLE
[Screen Reader-Command Palette]-Narrator focus is not in sync while navigating using arrow keys in suggestion list after searching for any command.

### ISSUE THREAD

**Issue by DasaradhaKamjula** [microsoft/terminal#7911]
**User Experience:**
Users who rely on screen reader will not be able get the control info if narrator focus is not in sync with arrow keys

**Test Environment:**
App: Windows Terminal Preview
Feature: Command Palette
Screen Reader: Narrator
OS: Win 10 2004 Build (19041.508)

**Repro Steps:**
1.Open Windows terminal Preview app
2.Open Command Palette using shortcut (Ctrl+shift+p)
3.Search for any command in the search field
4.Navigate using arrow key in the suggestion list.
5.Verify whether narrator focus moves onto suggestion list when user press down arrow key to move onto suggestions

**Actual**:
Narrator focus is not in sync while navigating using arrow keys in suggestion list after searching for any command.

**Expected:**
Narrator focus should be  in sync while navigating using arrow keys in suggestion list after searching for any command so that screen reader can read the focused control info.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

[Link to Attachment](https://microsoft-my.sharepoint.com/:v:/p/v-ramive/EWvcnRZbOvJDm7NnaVG2C8EBKfqjYIopMbnNHQknanjHCA?e=sw28SE)

**Comment by carlos-zamora** [comment]
I'm thinking that this might be able to be resolved by specifically handling the "up/down" arrow key events. We should try moving the focus programmatically to the highlighted item. Then, when you're typing any other key/character, we should move focus back to the text box. I'll play around with this today.

**Comment by Neurrone** [comment]
I just ran into this as well when using it with NVDA. The unintuitive thing to do is to press tab to move focus from the text field to the list of suggestions to have it read with u/down arrow.
